openlibrary/plugins/openlibrary/js/wikipedia_citation_button.js

```diff
@@ -0,0 +1,5 @@
+export function WikipediaCitationButtonPressed() {
+    $('#wikilink').click(function() {
+        $('#wikiselect').focus(function(){$(this).select();})
```

Comment:
Please take the opportunity to use the more modern `on` method instead of `focus` so that we can move towards a more updated version of jquery

e.g.

` $('#wikiselect').on('focus', function(){$(this).select();})`

Reply:
Done :)